Backpack/src/main/java/net/skyscanner/backpack/text/BpkText.kt

```diff
@@ -123,6 +133,20 @@ open class BpkText(
     if (drawableTint != null) {
       setDrawableTint(drawableTint.getColorForState(EMPTY_STATE_SET, Color.WHITE))
     }
+
+    val t = TypedValue()
+    if (context.theme.resolveAttribute(R.attr.bpkTextFont, t, true)) {
```

Comment:
We can simplify this by setting the default of `defStyleAttr` to be `R.attr.bpkTextStyle` and then just read those properties from the `a` typed array